module/actor/witcherActor.js

```diff
@@ -25,7 +24,266 @@ export default class WitcherActor extends Actor {
+    let luckTotalModifiers = this.getAllModifier("luck").totalModifiers;
+    let intDivider = this.getAllModifier("int").totalDivider;
+    let refDivider = this.getAllModifier("ref").totalDivider;
+    let dexDivider = this.getAllModifier("dex").totalDivider;
+    let bodyDivider = this.getAllModifier("body").totalDivider;
+    let spdDivider = this.getAllModifier("spd").totalDivider;
+    let empDivider = this.getAllModifier("emp").totalDivider;
+    let craDivider = this.getAllModifier("cra").totalDivider;
+    let willDivider = this.getAllModifier("will").totalDivider;
+    let luckDivider = this.getAllModifier("luck").totalDivider;
+    this.system.stats.int.modifiers.forEach(item => intTotalModifiers += Number(item.value));
+    this.system.stats.ref.modifiers.forEach(item => refTotalModifiers += Number(item.value));
+    this.system.stats.dex.modifiers.forEach(item => dexTotalModifiers += Number(item.value));
+    this.system.stats.body.modifiers.forEach(item => bodyTotalModifiers += Number(item.value));
+    this.system.stats.spd.modifiers.forEach(item => spdTotalModifiers += Number(item.value));
+    this.system.stats.emp.modifiers.forEach(item => empTotalModifiers += Number(item.value));
+    this.system.stats.cra.modifiers.forEach(item => craTotalModifiers += Number(item.value));
+    this.system.stats.will.modifiers.forEach(item => willTotalModifiers += Number(item.value));
+    this.system.stats.luck.modifiers.forEach(item => luckTotalModifiers += Number(item.value));
+
+    let stunTotalModifiers = this.getAllModifier("stun").totalModifiers;;
+    let runTotalModifiers = this.getAllModifier("run").totalModifiers;
+    let leapTotalModifiers = this.getAllModifier("leap").totalModifiers;
+    let encTotalModifiers = this.getAllModifier("enc").totalModifiers;
+    let recTotalModifiers = this.getAllModifier("rec").totalModifiers;
+    let wtTotalModifiers = this.getAllModifier("woundTreshold").totalModifiers;
+    let stunDivider = this.getAllModifier("stun").totalDivider;
+    let runDivider = this.getAllModifier("run").totalDivider;
+    let leapDivider = this.getAllModifier("leap").totalDivider;
+    let encDivider = this.getAllModifier("enc").totalDivider;
+    let recDivider = this.getAllModifier("rec").totalDivider;
+    let wtDivider = this.getAllModifier("woundTreshold").totalDivider;
+    this.system.coreStats.stun.modifiers.forEach(item => stunTotalModifiers += Number(item.value));
+    this.system.coreStats.run.modifiers.forEach(item => runTotalModifiers += Number(item.value));
+    this.system.coreStats.leap.modifiers.forEach(item => leapTotalModifiers += Number(item.value));
+    this.system.coreStats.enc.modifiers.forEach(item => encTotalModifiers += Number(item.value));
+    this.system.coreStats.rec.modifiers.forEach(item => recTotalModifiers += Number(item.value));
+    this.system.coreStats.woundTreshold.modifiers.forEach(item => wtTotalModifiers += Number(item.value));
```

Comment:
The same as above can also be applied here.

Reply:
https://github.com/witchertrpg-foundryvtt/TheWitcherTRPG/issues/61